index.html

```diff
@@ -43,6 +44,180 @@ <h1 class="text1">Hey There.<br>I'm Chris.</h1>
         <img src="illustration-header-mobile.svg" alt="some art" class="image">
       </main>
     </div>
-
+    <div class="page-2-container">
+      <header class="heading">
+        <h2 class="title-1" >My Recent Works</h> 
+        <nav class="line-pos">
+          <img src="line-top.svg" alt="line" class="line-top">
+        </nav>
+      </header>
+      <!-- work and about-->
+        <main class="body-container">
```

Comment:
kindly consider sticking tho the provided figma design as much as possible
- check the image below  👇 
![lll](https://user-images.githubusercontent.com/60146030/227745142-59ef8d9e-fef7-4679-9f6f-428d0e75cefc.PNG)
consider displaying all the works cards in the middle of the page decrease the padding too and this will make your project look more good

Reply:
since my last CR i have changed the following:
changed last slide issues
make body container display:flex
add button interactions hover,active

Reply:
- [ ] As the previous reviewer suggested, Kindly work on it. If it works properly on your browser you can record a screen video and attach it to your PR. :+1: 
[horizontal scrollbar.webm](https://user-images.githubusercontent.com/70483835/227746919-14ab4c87-ab7b-4636-a3f7-f435f07aa0ae.webm)


> kindly consider sticking tho the provided figma design as much as possible
> 
> * check the image below  point_down
>   ![lll](https://user-images.githubusercontent.com/60146030/227745142-59ef8d9e-fef7-4679-9f6f-428d0e75cefc.PNG)
>   consider displaying all the works cards in the middle of the page decrease the padding too and this will make your project look more good